README.md

```diff
@@ -18,16 +18,17 @@ background activity resumes normally.
 
 1. Ensure your Kubernetes is using `containerd` as the container runtime.
    - e.g. for minikube, add `--container-runtime containerd` to `minikube start` command:
-     `minikube start --kubernetes-version=v1.20.0 --container-runtime=containerd`
+     `minikube start --kubernetes-version=v1.20.7 --container-runtime=containerd`
```

Comment:
I wonder if we actually need a version any more since the latest stable (so minikube default) version is later than this